modules/ipi-install-network-requirements.adoc

```diff
@@ -160,5 +160,5 @@ You can reconfigure the control plane nodes to act as NTP servers on disconnecte
 
 [id='network-requirements-out-of-band_{context}']
 == Port access for the out-of-band management IP address
-
-The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180`.
+`
+The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management can communicate with the `baremetal` node during installation, the out-of-band management IP address must be granted access to port `6180` on the OpenShift control plane hosts, and port `80` on the bootstrap host.
```

Comment:
You could replace "OpenShift" with "`{product-title}`".

Reply:
Looking through the assembly, the only literal code references to "`baremetal`" in back-ticks are to the "`baremetal` network". You could replace "`baremetal` node" with "bare metal node" in this line, since I think the line is referring to the host type and not a code reference relating to the network component.

Reply:
@zaneb could you review this suggestion and make sure it's correct before I make the change? Thanks.